core/services/relay/evm/event_binding.go

```diff
@@ -227,32 +227,41 @@ func (e *eventBinding) getLatestValueWithFilters(
 		return err
 	}
 
-	fai := filtersAndIndices[0]
-	remainingFilters := filtersAndIndices[1:]
-
-	logs, err := e.lp.IndexedLogs(ctx, e.hash, e.address, 1, []common.Hash{fai}, confs)
+	// Create limiter and filter for the query.
+	limiter := query.NewLimitAndSort(query.CountLimit(1), query.NewSortBySequence(query.Desc))
+	filter, err := query.Where(
+		e.address.String(),
```

Comment:
Passing the address here isn't necessary and can be left empty. Since you're within the `evm` relay package here, you're able to use the `logpoller` filters directly and no key -> filter mappings are necessary. Putting an actual value here might just cause confusion later on.

Reply:
addressed in [`fa8056c` (#14096)](https://github.com/smartcontractkit/chainlink/pull/14096/commits/fa8056cc6dfefbc22f624051fdc00a9d34b5c3a0) and [`48aad52` (#14096)](https://github.com/smartcontractkit/chainlink/pull/14096/commits/48aad52ab566fdc2e526b90cde8f6a9a9fd7a5b3)